tests/v2/validation/nodescaling/README.md

```diff
@@ -53,18 +35,11 @@ These tests utilize Go build tags. Due to this, see the below examples on how to
 `gotestsum --format standard-verbose --packages=github.com/rancher/rancher/tests/v2/validation/nodescaling --junitfile results.xml -- -timeout=60m -tags=validation -v -run "TestNodeScaleDownAndUp/TestControlPlaneScaleDownAndUp"` \
 `gotestsum --format standard-verbose --packages=github.com/rancher/rancher/tests/v2/validation/nodescaling --junitfile results.xml -- -timeout=60m -tags=validation -v -run "TestNodeScaleDownAndUp/TestWorkerScaleDownAndUp"`
 
+Note: For these node replacing tests, you must have the following cluster configuration: 3 etcd, 2 cp, 3 worker.
```

Comment:
Is there a reason you moved to this format? It doesn't match our other readmes and imo would be easy to misss due to the change in structure.  We typically include this info as part of the yaml

Reply:
All of the other READMEs need to get refactored again. I didn't want to overbloat this PR so didn't do that here, but the other readmes shouldn't have the format they have now.

Reply:
spoke offline - lets make a separate ticket to handle this then so that there's consistency for all reame